### PR TITLE
MUR-7951

### PR DESCRIPTION
**Pull request by sauvainr**

### CODE REVIEW

**Comment by sauvainr** [comment]
Still need @johnlinvc  help to set the jenkins plugin for visualization.

File: Dockerfile [hunk 1]
```diff
@@ -0,0 +1,9 @@
+FROM java:8
+
+RUN wget http://mirror.nbtelecom.com.br/apache/jmeter/binaries/apache-jmeter-5.0.tgz
```
Comment (exjesper) [inline Dockerfile]: we should add https and SHA256 shasum verification

Comment (sauvainr) [reply Dockerfile]: Done, there is not mirror with valid ssl certificate, but as I check both the checksum & certificate that not necessary.

File: Jenkinsfile [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+    triggers {
+        cron('H 1 * * *')
+    }
+    parameters {
+      string(name: 'domain', defaultValue: 'perftest.apps.dev1.murano.exosite-dev.com')
+    }
+    stages {
+        stage('test') {
+            steps {
+                sh "DOMAIN=${params.domain} > murano.env"
```
Comment (exjesper) [inline Jenkinsfile]: docker compose actually supports variable substitution, https://docs.docker.com/compose/compose-file/#variable-substitution, but this is fine too

**Pull request merged**